pkg/awsManager/ebsManager.go

```diff
@@ -93,48 +93,48 @@ func ListVolumeForDeletion(client clientpkg.Client) []*string {
 	return ebsVolumesToBeDeleted
 }
 
-func DeleteEbsVolumes(client clientpkg.Client, ebsVolumesToBeDeleted []*string) error {
+func deleteEbsVolumes(client clientpkg.Client, ebsVolumesToBeDeleted []*string, logger logr.Logger) error {
```

Comment:
Why was this changed to unexported?

Reply:
because it is a helper function of CleanEbsVolumes()